include/common/graphic-helpers.h

```diff
@@ -1,4 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_COMMON_GRAPHIC_HELPERS_H
+#define LABWC_COMMON_GRAPHIC_HELPERS_H
```

Comment:
First of all: thanks for working on this.

The newly added include guards are slightly inconsistent with the other ones, the existing ones don't include the `COMMON` part whereas the new ones do. Personally I'd slightly prefer including the `COMMON` part but then we should do it consistently, e.g. the same for all subdirectories of `include/`. Or we leave them out everywhere and accept the potential (although minimal) risk of include guard clashes in the future.

Thoughts @johanmalm?

Reply:
I think we should leave them out (i.e. just base it on filename) and sort it case-by-case in future if there is an identifier clash.